docs/contributing/pull-request-guidelines/index.md

```diff
@@ -34,6 +34,7 @@ Use this workflow as a reference when you contribute to Autoware.
    - If you don't agree with a review comment, ask the reviewers for a rational reason.
      - The reviewers are obligated to make the author understand the meanings of each comment.
    - After you have done with the review comments, re-request a review to the reviewers and back to 6.
+     - Avoid using force push as much as possible so reviewers only see the differences. More precisely, at least keep a commit history up to the point of review because GitHub Web UI such as the the suggested change may require rebase to pass DCO CI.
```

Comment:
```suggestion
     - Avoid using force push as much as possible so reviewers only see the differences. More precisely, at least keep a commit history up to the point of review because GitHub Web UI such as the suggested change may require rebase to pass DCO CI.
```